x-pack/plugins/actions/server/lib/audit_events.ts

```diff
@@ -62,9 +63,9 @@ export function connectorAuditEvent({
     message,
     event: {
       action,
-      category: EventCategory.DATABASE,
-      type,
-      outcome: outcome ?? (error ? EventOutcome.FAILURE : EventOutcome.SUCCESS),
+      category: ['database'],
+      type: type ? [type] : undefined,
```

Comment:
I tested the Kibana filebeat module and confirmed that converting this to an array, while correct from an ECS standpoint, will still break existing filters, etc:

<img width="1148" alt="Screen Shot 2021-04-14 at 3 39 56 PM" src="https://user-images.githubusercontent.com/1608770/114784029-d9563600-9d37-11eb-8a8a-0694a1918ce1.png">

I'm working on an update to the module to correct this.

Reply:
Draft PR for the filebeat module: https://github.com/elastic/beats/pull/25101

(@legrego @thomheymann would appreciate any expertise you could offer on that) ❤️

Reply:
Thanks for testing how this gets displayed. I think we might need to check with the Observability team if they support arrays for `event.type` and `event.category` and how that should be rendered.